TSC-Charter.md

```diff
@@ -36,12 +36,15 @@ TSC by voluntary resignation or by a standard TSC motion. A standard TSC motion
 can be used to change a regular TSC member to a voting TSC member, or to change
 a voting TSC member to a regular TSC member.
 
-No more than one-fourth of the TSC voting members may be affiliated with the
-same employer. If change in TSC voting membership or a change of
-employment by a TSC voting member creates a situation where more than
-one-fourth of the TSC voting membership shares an employer, then the situation
-must be immediately remedied by the removal of voting member status from one or
-more TSC voting members affiliated with the over-represented employer(s).
+In order to ensure that hostile takeovers of the Node.js project are not possible,
+no company or other entity may influence more than one-fourth of the TSC voting membership.
```

Comment:
I think "influence" might be too broad? But maybe we want to be broad? Here's an alternative suggestion. Feel free to ignore it, though, if you think "influence" is better.

```suggestion
no company or other entity may employ or contract with more than one-fourth of the TSC voting membership.
```

Reply:
Oh, maybe "affiliated" is better than "influence" and we just need to rewrite the sentence a bit?

```suggestion
no more than one-fourth of the TSC voting membership may be affiliated with the same company or other entity. 
```